Makefile

```diff
@@ -17,7 +17,6 @@ functional-tests:
 
 clean clobber build deploy:
 	$(MAKE) -C chalice $@
-	$(MAKE) -C daemons $@
```

Comment:
Doesn't this remove the deployment of daemon as well?

Reply:
thank you